packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_navigation.spec.ts

```diff
@@ -20,18 +20,11 @@ context('multi-domain navigation', { experimentalSessionSupport: true, experimen
     })
   })
 
-  // FIXME: CypressError: Timed out after waiting `undefinedms` for your remote page to load.
-  // Your page did not fire its `load` event within `undefinedms`.
-  // You can try increasing the `pageLoadTimeout` value in `undefined` to wait longer.
-  // Browsers will not fire the `load` event until all stylesheets and scripts are done downloading.
-  // When this `load` event occurs, Cypress will continue running commands.
-  // at timedOutWaitingForPageLoad (webpack:///../driver/src/cy/commands/navigation.ts?:59:72)
-  // at eval (webpack:///../driver/src/cy/commands/navigation.ts?:586:16)
-  it.skip('.reload()', () => {
+  it('.reload()', () => {
```

Comment:
the visit and go tests don't work yet because visit in a switch to domain doesn't work yet